doc/ug/conf.py

```diff
@@ -128,6 +127,7 @@
 # Sphinx are currently 'default' and 'sphinxdoc'.
 #html_theme = 'basic'
 #html_theme = 'alabaster'
+html_theme = 'sphinx_rtd_theme'
```

Comment:
Same comment as above - it's another dependency that the user may not have. Is it really so much better than basic?